gradle.properties

```diff
@@ -48,7 +48,7 @@ micronautKafkaVersion=1.1.0.RC1
 micronautRabbitMQVersion=1.1.0.M2
 micronautNetflixVersion=1.0.4
 micronautGroovyVersion=1.1.0.RC1
-micronautDocsVersion=1.0.0.M5
+micronautDocsVersion=1.0.0.BUILD-SNAPSHOT
```

Comment:
Also for the beans.adoc to actually work it requires a new build of micronaut-docs due to this PR https://github.com/micronaut-projects/micronaut-docs/pull/8

Wasn't sure what to put here but just left it as 1.0.0.BUILD-SNAPSHOT since that is what was on master and that is what i used to test it locally.

Reply:
Not sure if this is something that would be worthwhile to put in CONTRIBUTING.md on what the pattern should be when you need to make a change in a dependent micronaut project. Or maybe this is not a common thing for most and it doesn't matter.